Dockerfile

```
@@ -0,0 +1,22 @@
+FROM openjdk:11-jre
+
+ARG MUSICBOT_VERSION
+
+WORKDIR /
+
+RUN pwd
+RUN ls -lrt
+
+ENV TOKEN harsh
+
+RUN echo "Building MusicBot version: $MUSICBOT_VERSION"
+
+# Download and rename the JAR file based on the provided version
+RUN curl -LJO "https://github.com/jagrosh/MusicBot/releases/download/$MUSICBOT_VERSION/JMusicBot-$MUSICBOT_VERSION.jar" \
+    && mv "JMusicBot-$MUSICBOT_VERSION.jar" JMusicBot.jar
```

It would be preferred to use the build artifacts from the previous job instead of fetching the release file like this. The files attached on a release are not necessarily the same as builds from source code.

Tip: with curl the mv step is also completely unnecessary since you can also save the file as a specified filename

```suggestion
RUN curl -o JMusicBot.jar -LJ "https://github.com/jagrosh/MusicBot/releases/download/$MUSICBOT_VERSION/JMusicBot-$MUSICBOT_VERSION.jar" 
```

---

Hi @jogerj 

Can you please help me with the change required to get the build form Artificats, I found this way more easier and understandable.